src/styles/components/form-elements/form-group/styles.less

```diff
@@ -29,6 +29,10 @@
           text-overflow: ellipsis;
           white-space: nowrap;
         }
+
+        .tooltip-wrapper {
+          max-height: 0.9rem;
```

Comment:
I would only try to use a variable if there is any available with the value `0.9rem`